### PR TITLE
fix: bulk api response type

### PR DESCRIPTION
**Pull request by therealansh**
Bulk Trigger returns list of trigger response

### CODE REVIEW

**Comment by therealansh** [comment]
@unicodeveloper @Cliftonz review please^

**Comment by unicodeveloper** [comment]
@mayorJAY please review

File: src/main/kotlin/api/EventsApi.kt [hunk 1]
```diff
@@ -21,7 +21,7 @@ interface EventsApi {
     suspend fun triggerEvent(@Body body: TriggerEventRequest): Response<ResponseWrapper<TriggerResponse>>
 
     @POST("$ENDPOINT/trigger/bulk")
-    suspend fun bulkTriggerEvent(@Body body: BulkTriggerEventRequest): Response<ResponseWrapper<TriggerResponse>>
+    suspend fun bulkTriggerEvent(@Body body: BulkTriggerEventRequest): Response<ResponseWrapper<List<TriggerResponse>>>
```
Comment (mayorJAY) [inline src/main/kotlin/api/EventsApi.kt:24]: Looks good but you need to modify the unit test based on your changes

Comment (therealansh) [reply src/main/kotlin/api/EventsApi.kt:24]: done

Comment (mayorJAY) [reply src/main/kotlin/api/EventsApi.kt:24]: LGTM @therealansh @unicodeveloper @Cliftonz

**Comment by therealansh** [comment]
Folks can we merge this and take this live asap i see there are few more commits which are not live yet?

**Review by Cliftonz — APPROVED** [review]

**Pull request merged**